src/previewSupport/handleElementHighlights.js

```diff
@@ -0,0 +1,54 @@
+import channel from './channel';
+
+function getOffsetFromBody(element) {
+  if (element === document.body) {
+    return {top: 0, left: 0};
+  }
+  const {top, left} = getOffsetFromBody(element.offsetParent);
+  return {top: top + element.offsetTop, left: left + element.offsetLeft};
+}
+
+function removeCovers() {
+  const highlighterElements =
+  window.document.querySelectorAll('.__popcode-highlighter');
+  for (let i = 0; i < highlighterElements.length; i++) {
+    const highlighterElement = highlighterElements[i];
+    highlighterElement.remove();
+  }
+}
+
+function addCovers(selector) {
+  const elements = document.querySelectorAll(selector);
+  for (let i = 0; i < elements.length; i++) {
+    const element = elements[i];
+    const cover = document.createElement('div');
+    document.body.appendChild(cover);
+    const rect = element.getBoundingClientRect();
+    let offset = {top: rect.top, left: rect.left};
+    if (element.offsetParent === null) {
+      cover.style.position = 'fixed';
+    } else if (element !== document.body ||
+      element !== document.documentElement) {
```

Comment:
The body seems to have an additional 8px of padding. So the box is off by a bit using getOffsetFromBody

Reply:
Actually looking again I think this condition will always evaluate to true, since it’s saying “if the element isn’t the body OR it isn’t the document root”? So maybe no need for the condition at all if it behaves correctly as written?